core/src/main/scala/akka/kafka/internal/TransactionalProducerStage.scala

```diff
@@ -111,32 +132,44 @@ private final class TransactionalProducerStageLogic[K, V, P](stage: Transactiona
     setHandler(stage.out, new OutHandler {
```

Comment:
These `setHandler` and `demandSuspended = false` should possibly become a method to stress they belong together.

Reply:
We have methods `resumeDemand` and `suspendDemand` now. Is this what you were thinking of?